compressor_test.go

```diff
@@ -14,7 +15,7 @@ func TestSnappyCompressor(t *testing.T) {
 	}
 
 	str := "My Test String"
-	//Test Encoding
+	//Test Encoding with Snappy
 	expected := snappy.Encode(nil, []byte(str))
```

Comment:
I'm not sure what this test does. It still tests with the original Snappy compressor?

Reply:
I wanted some test to check if it is indeed compatible but this test is not a good way of doing that, and I agree that S2 probably has such basic tests